src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -39,7 +39,7 @@ def dialog_content
         VBox(
           alternative_information,
           create_alternatives_table,
-          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          RichText(Id(:slaves), "<pre>#{@alternative.choices.first.slaves}</pre>"),
```

Comment:
and also is good idea to have always preselected first one? I expect that used one is selected, so if manually set to third element I expect it will be select. So what you can do is to select it in initialization and then call your `alternatives_handler`